spec/spec_helper_system.rb

```diff
@@ -0,0 +1,27 @@
+require 'rspec-system/spec_helper'
+require 'rspec-system-puppet/helpers'
+require 'rspec-system-serverspec/helpers'
+include RSpecSystemPuppet::Helpers
+
+RSpec.configure do |c|
+  # Project root
+  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+
+  # Enable colour
+  c.tty = true
+
+  c.include RSpecSystemPuppet::Helpers
+
+  # This is where we 'setup' the nodes before running our tests
+  c.before :suite do
+    # Install puppet
+    puppet_install
+
+    # Install modules and dependencies
+    puppet_module_install(source: proj_root, module_name: 'splunk')
+    shell('puppet module install puppet-archive --version 2.0.0')
```

Comment:
please take a look at the spec_helper_acceptance here:
https://github.com/voxpupuli/puppet-grafana/pull/61/commits/d09835a9453c50ea4c803a15fe2f44f1e57542e3

install_module_dependencies installs all modules from the metadata.json, install_module_from_forge is for additional dependencies.

Reply:
After #136 is merged i can do this.

Reply:
@bastelfreak Done.